Feature/updates (#59)
infinum/eightshift-docs: website/docs/basics/block-structure.md

### components
Components folder holds three files `heading-options.js`, `heading-editor.js`, and `heading-toolbars.js`. Each of these files represents a part of the Block Editor block that is used in the editor. We have separated options, editor, and toolbar into separate components for readability and reusing components in different projects.

### docs

> Docs folder contains everything that a block needs to be run inside the Storybook. This part is optional. But it will speed up your development time and provide you with the much needed documentation.

Reviewer: ```suggestion
Contains everything that a block needs to be run inside the Storybook. This part is optional. However, it will speed up your development time and provide you with the much needed documentation.
```

> - `story.js` - this is and Storybook entrypoint for your block. Basicity all blocks have the exact same story.

Reviewer: ```suggestion
- `story.js` - Storybook entrypoint for your block. Basicity all blocks have the exact same story.
```

> - `readme.mdx` - this is and readme document that describes your block and provides the documentation.

Reviewer: ```suggestion
- `readme.mdx` - readme document that describes your block and provides the documentation.
```
